lib/index.js

```diff
@@ -242,14 +266,14 @@ api.set = async ({
 };
 
 /**
- * Gets a token for an account.
+ * Gets tokens for `nonce` and token for other types for an account.
```

Comment:
Hmm, this seems problematic. Perhaps we need a `getAll` function for this. cc: @mattcollier, what would be your advice here? We had a previous pattern where there was only one token per `type` ... and now it's possible to have more than one. I would suspect that this might break other code elsewhere too.

Reply:
This may indicate we want IDs for tokens ... which is something we considered in the past: https://github.com/digitalbazaar/bedrock-authn-token/pull/7

Reply:
We could add IDs to tokens and make `get` optionally accept an `id` and if none is provided, just return the first one (for backwards compatibility) -- add a `getAll` to get all of them of a certain type. I'm not sure if anything depends on "overwriting"  a given token of a certain type (which wouldn't work for `nonce` anymore).

Reply:
+1 to adding a getAll function and ids to tokens.